src/domain/synthetics/markets/getBridigingInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import { Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import getBridgingOptionsForToken from "config/bridging";
+import { getChainName } from "config/chains";
+
+export default function getBridgingInfo(chainId: number, tokenSymbol?: string) {
```

Comment:
not sure that this code relatred to markets. Also there is jsx and not some maths.  Probably better to create component BridgingInfo

Reply:
Thanks, for the feedback. Updated